adafruit_tinylora/adafruit_tinylora.py

```diff
@@ -130,7 +130,7 @@ class TinyLoRa:
     _BUFFER = bytearray(2)
 
     # pylint: disable=too-many-arguments
-    def __init__(self, spi, cs, irq, rst, ttn_config, channel=None):
+    def __init__(self, spi, cs, irq, rst, ttn_config, channel=None, fport=1):
```

Comment:
In the Arduino Library the fport is passed as an argument to send_data  not to init ?
I am not familiar with its usage, but just noticed the difference.
Does it matter?

https://github.com/adafruit/TinyLoRa/blob/master/TinyLoRa.cpp#L610

Reply:
Agree with @jerryneedell on this point. I feel the frame port should be `0x01` by default and optionally set with a payload in the send function.

Specifically, I'd like to see `fport` defined as a kwarg in `send_data` here: (https://github.com/adafruit/Adafruit_CircuitPython_TinyLoRa/blob/master/adafruit_tinylora/adafruit_tinylora.py#L238) and set here (https://github.com/adafruit/Adafruit_CircuitPython_TinyLoRa/blob/master/adafruit_tinylora/adafruit_tinylora.py#L268).

Reply:
Is there any need for it to be saved as an attribute at all? should it just be passed in as fport and sent as 
```
lora_pkt[8] = fport
```
?

Reply:
Oh, my mistake. We'd want to change `send_packet` at the high-level, and then pass the frame port to `send_data` where it's actually set in the `lora_pkt`

 `def send_packet(self, lora_packet, packet_length, frame_port, timeout):`